app/controllers/image_sets_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ImageSetsController < ApiController
   before_filter :require_image_set, except: [:create, :index]
+  before_filter :require_image_set_ownership, only: [:destroy]
```

Comment:
this looks good. thanks. We can reuse this filter for the `update` command. I'll do that and add a test before merging.